src/scss/main.scss

```diff
@@ -111,17 +111,27 @@ body {
 }
 
 /* little shadow on top of the screen for no reasons */
-body::before {
+/* TinyMCE selector for body in preview excluded to prevent flashing */
```

Comment:
Thank you for this contribution !
It seems the patch is functioning :+1: . I was wondering, are those changes on the body / preview::before necessary ? i don't see what you mean by flashing
It seems to be displaying well without this first part, and prevents having this box shadow on the preview
![image](https://github.com/user-attachments/assets/e891914f-0a9f-4f11-bab7-ac874d621309)

Reply:
Hmm, it seems you're right. I believe I saw the text flashing before, but I can't reproduce that behavior now. I will revert some of these unnecessary style edits, including the `body::before` code snippet.